app/javascript/hooks/forms/admin/site_settings/useLinksForm.js

```diff
@@ -6,7 +6,7 @@ import { useCallback, useMemo } from 'react';
 
 export function useLinksFormValidation() {
   return useMemo(() => (yup.object({
-    value: yup.string().url('invalid_url'),
+    value: yup.string().url('forms.validations.url.invalid'),
```

Comment:
Your missing the translation here, what you have will just print `forms.validations.url.invalid`  as a literal string